sw/device/lib/crt/vectors.S

```diff
@@ -126,39 +126,32 @@ new vector table (which is at mtvec) */
 
 .section .text.vecs
 /* exception handling */
+.globl __no_irq_handler
 __no_irq_handler:
 	la a0, no_exception_handler_msg
 	jal ra, puts
 	j __no_irq_handler
 
 
-sw_irq_handler:
```

Comment:
we could keep the sw handler, in the future could be employed or even for people debugging